### ISSUE TITLE
Add ratings do the Minehut marketplace

### ISSUE THREAD

**Issue by aroze123** [Minehut/Meta#623]
### Feature request

**Checklist**
- [ ] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [ ] This is not a bug report, support request or plugin request
- [ ] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [ ] This issue does not contain any sensitive information 

**Platform**
Website

**Describe the feature**
Let people who've bought a product on the marketplace leave a review, with a star rating and some comments, or something along those lines.

**How the feature is useful**
There's a lot of items claiming to be stuff they just aren't, or over-exaggerating facts to a point where it's really misleading. Take Fox Anticheat for instance, detects almost nothing and falses like hell, and claims to be the best skripted anticheat on Minehut: [https://minehut.com/market/product/fox-anti-cheat](url)
[https://youtu.be/yiY8FwdTsb0](url)
Honestly not sure how this even got accepted.

These products don't seem to be reviewed by staff, at least not well, clearly. So community reviews could really help,

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by godModeD** [comment]
Very cool feature.

**Comment by Shlba** [comment]
A good idea from aroze?! +1

**Comment by quick007** [comment]
This was meant to be included with v2 release, what happened?

**Comment by ptrlol** [comment]
I agree with this - It's needed because some items you'd purchase are not what they claim to be and it's just a loss of credits.

**Comment by ReportCardsMC** [comment]
> This was meant to be included with v2 release, what happened?

@quick007 V2 Hasn't been fully implemented yet iirc

But yes +1 to this, though it is already planned :)

**Comment by drtshock** [comment]
This was added.
